### DISCUSSION TITLE
fun_story/

### DISCUSSION THREAD

**Discussion by giscus** [Micraow/chat#3] [Announcements]
# fun_story/

元宵节那天，不小心让电脑进了水，拿到了电脑城去维修，坑蒙拐骗店员去了解了archlinux，还解锁了“特别优惠” - Micraow - Peng's Blog

https://pengs.top/fun_story/

**Comment by c-jeremy** [comment]
哈哈哈哈！小彭还是和之前一样的有趣（不过身为修电脑的，只知道Windows确实有点离谱🤣

**Reply by Micraow** [reply]
唉，可能还是太年轻了
